pkgs/development/libraries/qt-4.x/4.8/default.nix

```diff
@@ -185,6 +192,6 @@ stdenv.mkDerivation rec {
     description = "A cross-platform application framework for C++";
     license     = licenses.lgpl21Plus; # or gpl3
     maintainers = with maintainers; [ lovek323 phreedom sander urkud ];
-    platforms   = platforms.unix;
+    platforms   = platforms.unix ++ platforms.darwin;
   };
```

Comment:
This change is not needed: `platforms.unix` already includes `platforms.darwin`. See the definition at https://github.com/NixOS/nixpkgs/blob/master/lib/platforms.nix#L21